src/neptune/internal/utils/images.py

```diff
@@ -196,36 +197,57 @@ def _image_content_to_html(content: bytes) -> str:
     return "<img src='data:image/png;base64," + str_equivalent_image + "'/>"
 
 
-def _get_numpy_as_image(array):
+def _get_numpy_as_image(array: np.ndarray, autoscale: bool) -> bytes:
```

Comment:
I'll get back to you as we probably can get rid of the `autoscale` param

Reply:
@AleksanderWWW - [0,255]

Reply:
Closing as we should proceed with the `autoscale` param just in case. @szysad had some examples that could break our autodetection that were consistent with my initial thoughts.